mzLib/Proteomics/PSM/PsmTsvHeader_Glyco.cs

```diff
@@ -2,10 +2,10 @@
 {
     public enum LocalizationLevel
     {
-        Level1,
-        Level1b,
-        Level2,
-        Level3
+        Level1 = 0,
```

Comment:
levels are defined in lei's paper. i don't think we can just change them. 
![image](https://github.com/user-attachments/assets/becefbda-7239-4981-9c4e-c88d2e620d97)

Reply:
Not changing them. We had this enum in duplicated in the code. I deleted one copy and just added the explicit integer values here. This is so anyone reading to code can easily understand the order of beset-> worst localization level